parity-clib/examples/cpp/main.cpp

```diff
@@ -109,16 +88,17 @@ int parity_rpc_queries(void* parity) {
 		return 1;
 	}
 
-	Callback cb { .type = CALLBACK_RPC, .counter = rpc_queries.size() };
+	Callback cb { .type = CALLBACK_RPC };
```

Comment:
This was a non-standard extension to C++ last I checked, but this may have changed in C++17.

Reply:
yeah, you are right `designated initializers` is a C++20 thing

Reply:
[C++ 11](https://github.com/paritytech/parity-ethereum/blob/master/parity-clib/examples/cpp/CMakeLists.txt#L4) but might possible to bump it to c++17.

I think it is a `CI` related question